ister_test.py

```diff
@@ -468,6 +468,11 @@ def run_command_good():
     ister.run_command("true")
 
 
+def run_command_with_env():
+    """run_command with environment variable passed"""
+    ister.run_command("true", environ=os.environ)
```

Comment:
These run command tests are not very good =(. I feel bad about that as they really should be mocked but more to validate correctness but I didn't do that yet. So given that this test is fine but it really doesn't tell you if it is using the environment or not from the test result.